packages/collabswarm/src/collabswarm-document.ts

```diff
@@ -35,6 +35,8 @@ export class CollabswarmDocument<DocType, ChangesType, ChangeFnType, MessageType
     private readonly _provider: CRDTProvider<DocType, ChangesType, ChangeFnType, MessageType>,
     private readonly _changesSerializer: ChangesSerializer<ChangesType>,
     private readonly _messageSerializer: MessageSerializer<MessageType>,
+    private _read_ACL: {},
+    private _write_ACL: {}
```

Comment:
```suggestion
    private _writeACL: {[name: string]: CryptoKey}  // Or something like that.
```

The current typedef (`{}` as a type means specifically an empty dictionary only) prohibits adding any keys.